### ISSUE TITLE
Console errors when navigating between articles

### ISSUE THREAD

**Issue by 12** [bbc/simorgh#3150]
**Describe the bug**
When clicking an internal link to a new article, these errors are thrown and appear in console:
```
Warning: Failed prop type: The prop `data.dials` is marked as required in `PageWrapperContainer`, but its value is `undefined`.
Warning: Failed prop type: The prop `data.dials` is marked as required in `DataContainer`, but its value is `undefined`.
``` 

**To Reproduce**

1. Run `npm run dev`
2. Go to http://localhost.bbc.com:7080/news/articles/c0g992jmmkko
3. Click on the link to another article `placed on the tomb of the unknown warrior`
4. Open console
5. See error

**Expected behavior**
When navigating through articles, I should not see any console errors

**Additional context**

**Comment by chris-hinds** [comment]
Not sure if we are still seeing these console warning, so double check before starting any work.

However if we seeing the warning, anything todo with Dials can be safely removed as Simorgh no longer uses cosmos dials.

**Comment by pharingee** [comment]
I'm not able to reproduce this error. Like @hindsc52 stated earlier the cosmos dials have been removed

**Comment by pharingee** [comment]
There are a couple of tests where dials are still referenced, although these references can be removed safely:

https://github.com/bbc/simorgh/blob/latest/src/app/containers/Error/index.test.jsx#L11
https://github.com/bbc/simorgh/blob/latest/src/app/containers/PageHandlers/withData/index.test.jsx#L54

@hindsc52  Should we open a new issue for that, or should we reference this ticket in that [PR](https://github.com/bbc/simorgh/pull/4166).
